tests/test_raft.rs

```diff
@@ -529,41 +556,275 @@ fn test_progress_paused() {
 }
 
 #[test]
-fn test_leader_election() {
+fn test_leader_election_no_pre_vote() {
```

Comment:
I think it's better to keep the original name and rename `test_leader_election` to `test_leader_election_with_config`.